programs/ssl/ssl_server2.c

```diff
@@ -1791,7 +1791,7 @@ int main( int argc, char *argv[] )
             unsigned char *larger_buf;
 
             ori_len = ret;
-            extra_len = ssl_get_bytes_avail( &ssl );
+            extra_len = (int) ssl_get_bytes_avail( &ssl );
```

Comment:
I think it is better to change `extra_len` to `size_t` type, and thus remove the need for casting